catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveCatalogOperations.java

```diff
@@ -135,11 +152,87 @@ public void initialize(Map<String, String> conf) throws RuntimeException {
     // and gravitinoConfig will be passed to Hive config, and gravitinoConfig has higher priority
     mergeConfig.forEach(hadoopConf::set);
     hiveConf = new HiveConf(hadoopConf, HiveCatalogOperations.class);
+    UserGroupInformation.setConfiguration(hadoopConf);
+
+    initKerberosIfNecessary(conf, hadoopConf);
 
     this.clientPool =
         new CachedClientPool(getClientPoolSize(conf), hiveConf, getCacheEvictionInterval(conf));
   }
 
+  private void initKerberosIfNecessary(Map<String, String> conf, Configuration hadoopConf) {
+    if (UserGroupInformation.AuthenticationMethod.KERBEROS
+        == SecurityUtil.getAuthenticationMethod(hadoopConf)) {
+      try {
+        File keytabTemporaryDirectory = new File("tmp");
```

Comment:
I would suggest to use name of "keytabs", is it better?

Reply:
Fixed.